modules/navier_stokes/tests/cnsfv/2d_bump_impl_ref3.i

```diff
@@ -0,0 +1,418 @@
+  slope_limiting = lslope
+  boundary_list = '1 2 3 4'
+  boundary_condition_user_object_list = 'inflow_bcuo
+                                         outflow_bcuo
+                                         slip_bcuo
+                                         slip_bcuo'
+  infinity_density = 1.
+  infinity_x_velocity = 0.5
+  infinity_pressure = 0.71428571428571428571
+  implicit = true
+[]
+############################################################
+[Mesh]
+  file = SmoothBump_quad_ref3_Q1.msh
+[]
+############################################################
+[Problem]
+ kernel_coverage_check = false
+[]
+############################################################
+[Functions]
+[]
+############################################################
+[Modules]
+  [./FluidProperties]
+    [./fp]
+      type = IdealGasFluidProperties
+      gamma = 1.4
+      R = 0.71428571428571428571
+    [../]
+  [../]
+[]
+############################################################
+[UserObjects]
+  [./slip_bcuo]
+    execute_on = 'linear'
+    type = CNSFVSlipBCUserObject
+  [../]
+
+  [./inflow_bcuo]
+    execute_on = 'linear'
+    type = CNSFVRiemannInvariantBCUserObject
+  [../]
+
+  [./outflow_bcuo]
+    execute_on = 'linear'
+    type = CNSFVRiemannInvariantBCUserObject
+  [../]
+
+  [./rslope]
+    execute_on = 'linear'
+    type = CNSFVLeastSquaresSlopeReconstruction
+  [../]
+
+  [./lslope]
+    execute_on = 'linear'
+    type = CNSFVNoSlopeLimiting
+  [../]
+
+  [./riemann]
+    execute_on = 'linear'
+    type = CNSFVHLLCInternalSideFlux
+  [../]
+
+  [./inflow_bc]
+    execute_on = 'linear'
+    type = CNSFVRiemannInvariantBoundaryFlux
+    bc_uo = 'inflow_bcuo'
+  [../]
+
+  [./outflow_bc]
+    execute_on = 'linear'
+    type = CNSFVRiemannInvariantBoundaryFlux
+    bc_uo = 'outflow_bcuo'
+  [../]
+
+  [./slip_bc]
+    execute_on = 'linear'
+    type = CNSFVHLLCSlipBoundaryFlux
+    bc_uo = 'slip_bcuo'
+  [../]
+[]
+############################################################
+[Variables]
+  [./rho]
+  [../]
+
+  [./momx]
+  [../]
+
+  [./momy]
+  [../]
+
+  [./rhoe]
+  [../]
+[]
+############################################################
+[AuxVariables]
+  [./mach]
+  [../]
+
+  [./pres]
+  [../]
+[]
+############################################################
+[ICs]
+  [./rho_ic]
+    variable = 'rho'
+    type = ConstantIC
+    value = 1.
+  [../]
+
+  [./rhou_ic]
+    variable = 'momx'
+    type = ConstantIC
+    value = 0.5
+  [../]
+
+  [./rhov_ic]
+    variable = 'momy'
+    type = ConstantIC
+    value = 0.
+  [../]
+
+  [./rhoe_ic]
+    variable = 'rhoe'
+    type = ConstantIC
+    value = 1.91071428571428571429
+  [../]
+
+  [./mach_ic]
+    type = CNSFVMachIC
+    variable = 'mach'
+  [../]
+
+  [./pres_ic]
+    type = CNSFVPressureIC
+    variable = 'pres'
+  [../]
+[]
+############################################################
+[Kernels]
+  #### Time derivative of mass
+  [./time_rho]
+    implicit = true
```

Comment:
This is most likely copy and paste from an explicit solve, `implicit = true` by default, no need for it.